conda-forge.yml

```
@@ -9,6 +9,6 @@ github:
   branch_name: main
   tooling_branch_name: main
 provider:
-  linux_aarch64: default
-  linux_ppc64le: default
-test: native_and_emulated
+  linux_aarch64: azure
+  linux_ppc64le: azure
+test: native
```

Let's change this back to the value here:
```suggestion
test: native_and_emulated
```
I misunderstood a problem I was having elsewhere and now I know this is okay.

---

We'll also need a rerender after that change.